<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWithButtons</class>
 <widget class="QDialog" name="ListWithButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_calculate">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QListWidget" name="list"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
